### PR TITLE
Enhancements to Filter Component of InboxSearchComposer

### PR DESCRIPTION
**Pull request by nipunarora-eGov**
-> Added three new components that can be used in inbox filters (or anywhere else) -> They are : MultiSelectDropdowns , LocationData dropdowns, WorkflowStatus filters

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/molecules/WorkflowStatusFilter.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React,{Fragment} from 'react'
+import CheckBox from "../atoms/CheckBox"
+import { Loader } from '../atoms/Loader';
+
+const WorkflowStatusFilter = ({props,t,populators,formData}) => {
+
+    const { data, isLoading } = Digit.Hooks.useApplicationStatusGeneral({ businessServices: ["estimate-approval"] }, {});
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/molecules/WorkflowStatusFilter.js:7]: Send business service as props

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Estimate/src/pages/employee/EstimateInbox.js [hunk 2]
```diff
@@ -2,11 +2,11 @@ import React from "react";
 import { useTranslation } from "react-i18next";
 import { Header, InboxSearchComposer } from "@egovernments/digit-ui-react-components";
 import inboxConfig from "../../configs/inboxConfig";
-
+import inboxConfigMukta from "../../configs/inboxConfigMukta";
 const EstimateInbox = () => {
     const { t } = useTranslation();
 
-    const configs = inboxConfig();
+    const configs = inboxConfigMukta(t);
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/Estimate/src/pages/employee/EstimateInbox.js:9]: Integrate with mdms config

**Review by jagankumar-egov — CHANGES_REQUESTED** [review]

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/Estimate/src/pages/employee/EstimateInbox.js (hunk 2 above)
Comment (nipunarora-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/Estimate/src/pages/employee/EstimateInbox.js:9]: This PR is for enhancements only . Will give a separate pr for inbox

File: frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/molecules/WorkflowStatusFilter.js (hunk 1 above)
Comment (nipunarora-eGov) [reply frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/molecules/WorkflowStatusFilter.js]: Added BusinessService in populators

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**